pure-pso/README.md

```diff
@@ -226,7 +226,9 @@ The following table lists the configurable parameters and their default values.
 | `images.database.cockroachOperator.tag`        | The image tag to pull                                                                                                                                      | `v1.0.1      `                                |
 | `images.database.deployer.name`                | The image name of the cockroach db deployer                                                                                                                | `purestorage/dbdeployer           `           |
 | `images.database.deployer.pullPolicy`          | Image pull policy                                                                                                                                          | `Always      `                                |
-| `images.database.cockroachOperator.tag`        | The image tag to pull                                                                                                                                      | `v1.0.1      `                                |
+| `images.database.deployer.tag`                 | The image tag to pull                                                                                                                                      | `v1.0.1      `                                |
+| `images.database.psctl.name`                   | The image name of PSCTL                                                                                                                                    | `purestorage/psctl                `           |
```

Comment:
The table does not look right.  should have "`" after purestorage/psctl

Reply:
It's there, just a little bit further down the line. I'm not sure why this trend started but I'm following it with the table for now, at some point we added extra spaces in the `code block`.